beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestCatalogValidation.java

```diff
@@ -67,6 +65,24 @@ public void testUploadCatalog() throws Exception {
     	}
     }
     
+    @Test(groups = "slow", description = "https://github.com/killbill/killbill/issues/1465")
+    public void testUploadCatalogPlanValidation() throws Exception {
+	try {
+	    uploadCatalog("CatalogValidation-v2.xml");
+	    assertListenerStatus();
```

Comment:
Is this for testing #1481, if not what are we testing?

Reply:
Yes you are right. I have reverted this change.